generated/nidcpower/_attributes.py

```diff
@@ -115,10 +115,10 @@ class AttributeSessionReference(Attribute):
     def __get__(self, session, session_type):
         # Import here to avoid a circular dependency when initial import happens
         from nidcpower.session import SessionReference
-        return SessionReference(session._get_attribute_vi_int32(self._attribute_id))
+        return SessionReference(session._get_attribute_vi_session(self._attribute_id))
```

Comment:
`_get_attribute_vi_session` doesn't actually _exist_ in `nidcpower.Session` nor in a few other drivers. I'm not sure how much we care. I suppose that's a question for a different PR #1039

Reply:
Yes.